tests/integration/steps/integration_steps.go

```diff
@@ -133,43 +132,70 @@ func DeployClusterWithValues(t *testing.T, namespace, options, customValues stri
 			"medusa.storage_properties.host": fmt.Sprintf("%s.minio.svc.cluster.local", serviceName),
 		}
 	}
+	helmValues["cassandra.datacenters[0].size"] = strconv.Itoa(nodes)
+	helmValues["cassandra.datacenters[0].name"] = datacenterName
 	deployCluster(t, namespace, customValues, helmValues)
 }
 
-func DeployClusterWithValuesAndHeapSettings(t *testing.T, namespace, options, cassandraHeap, stargateHeap, customValues string) {
-	log.Printf("I can deploy a cluster with %s options, %s Cassandra Heap and %s Stargate heap using the %s values", options, cassandraHeap, stargateHeap, customValues)
-
-	splitOptions := strings.Split(options, "-")
-	medusaEnabled := "true"
-	reaperEnabled := "true"
-	monitoringEnabled := "true"
+func ScaleClusterWithValues(t *testing.T, namespace, options, customValues string, nodes int) {
```

Comment:
`ScaleClusterWithValues` looks identical to `DeployClusterWithValues` except for the last line where it calls `scaleCluster` instead of `scaleCluster`. Maybe we could refactor the duplicate code?

Reply:
I can probably remove some code duplication indeed.

Reply:
Since most of the code is exactly the same, I deleted the `ScaleCluster***` functions and added an `upgrade` boolean as argument for `DeployCluster***` to distinguish running `helm install` and `helm upgrade`.

Good recommendation 👍